edgedb/ai/__init__.py

```diff
@@ -0,0 +1,32 @@
+#
+# This source file is part of the EdgeDB open source project.
+#
+# Copyright 2024-present MagicStack Inc. and the EdgeDB authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from .types import AIOptions, ChatParticipantRole, Prompt, QueryContext
+from .core import create_AI, EdgeDBAI
+from .core import create_async_AI, AsyncEdgeDBAI
+
+__all__ = [
+    "AIOptions",
+    "ChatParticipantRole",
+    "Prompt",
+    "QueryContext",
+    "create_AI",
```

Comment:
Lowercase spelling seems better to me:

```suggestion
    "create_ai",
```